app/ios/Runner/Info.plist

```diff
@@ -22,8 +22,45 @@
 	<string>$(FLUTTER_BUILD_NAME)</string>
 	<key>CFBundleSignature</key>
 	<string>????</string>
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Viewer</string>
+			<key>CFBundleURLName</key>
+			<string>global.acter.url.matrix</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>matrix</string>
+			</array>
+		</dict>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Editor</string>
+			<key>CFBundleURLName</key>
+			<string>global.acter.urls.acter</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>acter</string>
+			</array>
+		</dict>
```

Comment:
that other wasn't the correct format for `acter:` and `matrix:` URIs, this now is .